### PR TITLE
Continue tour

### PR DESCRIPTION
**Pull request by milleruntime**
Exercises added for:
Auths
data model
ranges and splits
batch scanner

### CODE REVIEW

File: tour/ranges-splits.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+As your data grows larger, Accumulo will split tables across multiple servers called Tablet Servers.  
+By default a table will get split on row boundaries, guaranteeing an entire row to be on one Tablet Server.  We have the ability to 
+tell Accumulo were to split tables by setting split points. This is done using _addSplits_ in the [TableOperations] API.  The image below 
+demonstrates how Accumulo splits data.  
+
+![data distribution]({{ site.url }}/images/docs/data_distribution.png)
+
+There are many useful administrative methods in [TableOperations] so take a minute to look through the API. Here are some terms specific to Accumulo:
```
Comment (keith-turner) [inline tour/ranges-splits.md]: This is good content, but it feels out of place here.

File: tour/ranges-splits.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Ranges and Splits
+---
+
+A Range is a specified group of Keys. There are many different ways to create a Range.  Here are a few examples:
+```java
+new Range(Key startKey, Key endKey)  // Creates a range from startKey inclusive to endKey inclusive.
+new Range(CharSequence row)  // Creates a range that covers an entire row.
```
Comment (keith-turner) [inline tour/ranges-splits.md]: Its odd that Jekyll renders this `Range` as red but not the one on the line before.

File: tour/authorizations.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Authorizations
+---
+An Authorization is an authority granted to a User that allows them to read certain data. A column in Accumulo can have 
+a visibility attached to it, limiting access to the data of that column. Only users who have an authorization equal to 
+the visibility will be able to read the data in that column.
```
[the remark this answers is not in the document]
Comment (milleruntime) [reply tour/authorizations.md]: That does sound better but I was trying to make it clear that authorization = visibility.  This was something that tripped me up as a beginner, they have different names for different reasons but it isn't clear anywhere that it is a simple string compare.

[the remark this answers is not in the document]
Comment (keith-turner) [reply tour/authorizations.md]: One is a set and one is an expression.  Could say a column visibility is a boolean expression that is evaluated using a scanners authorizations.  If it evaluates to true, then the data is visible.

File: tour/authorizations.md [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
-An Authorization is an authority granted to a User that allows them to read certain data. A column in Accumulo can have 
-a visibility attached to it, limiting access to the data of that column. Only users who have an authorization equal to 
-the visibility will be able to read the data in that column.
+Authorizations are a set of Strings that enable a user to read protected data. A column visibility is a boolean expression 
+that is evaluated using the authorizations provided by a scanner. If it evaluates to true, then the data is visible. 
+
+For example:
+* Bob has authorizations = { IT, User }
+* Tina has authorizations = { Admin, IT, User }
+* Row1:A:B has Visibility = { Admin && IT && User }
```
Comment (keith-turner) [inline tour/authorizations.md]: Could change `A:B` to `family1:qualifier1`.   This makes the line longer, but if may make it more clear what the fields are.

File: tour/authorizations.md [hunk 5]
```diff
@@ -1,9 +1,15 @@
...
 title: Authorizations
 ---
-An Authorization is an authority granted to a User that allows them to read certain data. A column in Accumulo can have 
-a visibility attached to it, limiting access to the data of that column. Only users who have an authorization equal to 
-the visibility will be able to read the data in that column.
+Authorizations are a set of Strings that enable a user to read protected data. A column visibility is a boolean expression 
+that is evaluated using the authorizations provided by a scanner. If it evaluates to true, then the data is visible. 
+
+For example:
+* Bob has authorizations = { IT, User }
```
Comment (keith-turner) [inline tour/authorizations.md]: This example is nice.   Could say `Bob scans with authorizations`

File: tour/data-model-code.md [hunk 6]
```diff
@@ -35,31 +38,21 @@ title: Data Model Code
...
-Value:Dick Grayson
-Key:id0002 hero:wearsCape? [] 1510084285325 false
-Value:true
-Key:id0003 villain:alias [] 1510084285325 false
-Value:Joker
-Key:id0003 villain:name [] 1510084285325 false
-Value:Unknown
-Key:id0003 villain:wearsCape? [] 1510084285325 false
-Value:false
+Key : id0001 hero:alias [] 1511306370025 false            Value : Batman
```
Comment (keith-turner) [inline tour/data-model-code.md]: The timestamp and delete boolean makes the output noisy.  However trying to omit would make the code noisy/longer and harder to understand.  I can't think of any way to improve this though.

File: tour/ranges-splits.md (hunk 1 above)
Comment (milleruntime) [reply tour/ranges-splits.md]: I agree. I was trying to point out the more useful API classes and the best places to start.  Maybe we could do that in another exercise

File: tour/ranges-splits.md (hunk 2 above)
Comment (milleruntime) [reply tour/ranges-splits.md]: I was going to merge the content I had and then let @mikewalch clean it up.  Maybe he can figure out why.

**Comment by milleruntime** [comment]
I am merging this up to the branch so others can continue work on the tour.

**Pull request merged**